### ISSUE TITLE
doesn't show in keyboard list switcher

### ISSUE THREAD

**Issue by Caleb-D** [urob/colemak-win#1]
![image](https://user-images.githubusercontent.com/35500148/231568269-4fd82428-b3c6-4ef5-9aba-d7128789b51c.png)

**Comment by urob** [comment]
Does it show up when you go to Language preferences > Preferred languages > English > Options > Keyboards? If not, you can add it there using "Add a keyboard" (after installing the layout using the "setup.exe" from this repo)

**Comment by Caleb-D** [comment]
yes, it shows up there, and I can select it to make it the default, but it is difficult not being able to switch back and forth (using the switcher from the taskbar) when you are trying to learn a new layout.  I think I've noticed this problem with other layouts so may not be specific to this colemak layout.

**Comment by urob** [comment]
You don't have to make it the default, but you'll have to add it to the list of keyboards installed for your *selected* language in order to show up in the language bar (you can add more than one).

<img width="611" alt="Screenshot 2023-04-12 170606" src="https://user-images.githubusercontent.com/978080/231584898-50dd1573-329c-4da5-9a0a-8abcb9534b6f.png">

**Comment by Caleb-D** [comment]
Weird.  The selector doesn't show up if I remove the dvorak layout and only have what you show:
![image](https://user-images.githubusercontent.com/35500148/231585533-b9045a98-ba7b-4b31-80e1-b7ed25f83bf5.png)

**Comment by Caleb-D** [comment]
Wonder if there's a bug in the version of Win10 I have.
Edition	Windows 10 Enterprise
Version	21H2
Installed on	‎6/‎13/‎2022
OS build	19044.2728
Experience	Windows Feature Experience Pack 120.2212.4190.0
